content/en/docs/collector/architecture.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Architecture
+weight: 28
+# prettier-ignore
+cSpell:ignore: # TODO: Add keywords to ignore.
+---
+
+OpenTelemetry Collector is an executable that can receive telemetry data,
+optionally process it, and export it further.
+
+The Collector supports several popular open source protocols for receiving and
+sending telemetry data as well as offering a pluggable architecture for adding
+more protocols.
+
+Data receiving, processing, and exporting are done using
+[Pipelines](#pipelines). The Collector can be configured to have one or more
+pipelines. Each pipeline includes the following:
+
+- A set of [Receivers](#receivers) that receive the data.
+- A series of optional [Processors](#processors) that get the data from
+  receivers and process it.
+- A set of [Exporters](#exporters) which get the data from processors and send
+  it outside the Collector.
+
+The same receiver can be included in multiple pipelines and multiple pipelines
+can include the same exporter.
+
+## Pipelines
+
+A pipeline defines a path that data follows in the Collector: from reception, to
+processing (or modification), and finally to export.
+
+Pipelines can operate on three telemetry data types: traces, metrics, and logs.
+The data type is a property of the pipeline defined by its configuration.
+Receivers, processors, and exporters used in a pipeline must support the
+particular data type otherwise `ErrDataTypeIsNotSupported` will be reported when
+the configuration is loaded. A pipeline can be depicted the following way:
+
+```mermaid
+---
+title: Pipeline
+---
+flowchart LR
+    A(Receiver 1) --> D[Processor 1]
+    B(Receiver 2) --> D
+    C(Receiver N) --> D
+    D --> E[Processor 2]
+    E --> F[Processor N]
+    F --> G((fan out))
+    G --> H[[Exporter 1]]
+    G --> I[[Exporter 2]]
+    G --> J[[Exporter N]]
+
+    classDef default fill:#e3e8fc,stroke:#4f62ad 
+```
+
+Pipelines can have one or more receivers. Data from all receivers is pushed to
+the first processor, which processes the data and then pushes it to the next
+processor (or a processor may drop the data if it is a “sampling” processor, for
+example), and so on until the last processor in the pipeline pushes the data to
+the exporters. Each exporter gets a copy of each data element. The last
+processor uses a `fanoutconsumer` to fan out the data to multiple exporters.
+
+The pipeline is constructed during Collector startup based on pipeline
+definition in the configuration.
+
+A pipeline configuration typically looks like this:
+
+```yaml
+service:
+  pipelines: # section that can contain multiple subsections, one per pipeline
+    traces: # type of the pipeline
+      receivers: [otlp, jaeger, zipkin]
+      processors: [memory_limiter, batch]
+      exporters: [otlp, jaeger, zipkin]
```

Comment:
```suggestion
      receivers: [otlp, zipkin]
      processors: [memory_limiter, batch]
      exporters: [otlp, zipkin]
```

`jaeger` is deprecated since there is native OTLP support, so we might consider removing them here.